gtfs/spec/en/reference.md

```diff
@@ -401,10 +402,13 @@ File: **Optional**
 
 Trip planners normally calculate transfer points based on the relative proximity of stops in each route. For potentially ambiguous stop pairs, or transfers where you want to specify a particular choice, use transfers.txt to define additional rules for making connections between routes.
 
+**Note**: Transfers between stops more than 2 km apart are discouraged.
+
 |  Field Name | Required | Details |
 |  ------ | ------ | ------ |
 |  from_stop_id | **Required** | The **from_stop_id** field contains a stop ID that identifies a stop or station where a connection between routes begins. Stop IDs are referenced from the [stops.txt](#stopstxt) file. If the stop ID refers to a station that contains multiple stops, this transfer rule applies to all stops in that station. |
 |  to_stop_id | **Required** | The **to_stop_id** field contains a stop ID that identifies a stop or station where a connection between routes ends. Stop IDs are referenced from the [stops.txt](#stopstxt) file. If the stop ID refers to a station that contains multiple stops, this transfer rule applies to all stops in that station. |
+|| | Note: **from_stop_id** and **to_stop_id** should not refer to station-entrance type stops. |
```

Comment:
If you want to clear things up and explictly mark that it should not refer to one location type, I would also like to know more about the other location types possible ;)